graphs/possible_bipartition.py

```diff
@@ -1,12 +1,36 @@
 # Can be used for BFS
-from collections import deque 
+from collections import deque
+
 
 def possible_bipartition(dislikes):
-    """ Will return True or False if the given graph
-        can be bipartitioned without neighboring nodes put
-        into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+    """Will return True or False if the given graph
+    can be bipartitioned without neighboring nodes put
+    into the same partition.
+    Time Complexity: ?
+    Space Complexity: ?
     """
-    pass
 
+    if not dislikes:
+        return True
+
+    groups = [None] * len(dislikes)
+    group_1 = 0
+    group_2 = 1
```

Comment:
Consider making all caps, as you use these as constants
```suggestion
    GROUP_1 = 0
    GROUP_2 = 1
```